docs/analyzer-rules/AZFW0005.md

```diff
@@ -0,0 +1,27 @@
+# AZFW0005: Multiple Azure Functions Binding Attributes grouped together
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW0005|
+| **Category** |[FunctionMetadataGeneration]|
```

Comment:
What is the purpose of the square brackets around FunctionMetadataGeneration?

Reply:
Just following the convention set by the other analyzer rules ([example](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/docs/analyzer-rules/AZFW0001.md)). Not sure if it actually does anything. 

cc: @kshyju @fabiocav are the other two contributors to the analyzer rules. They may have more context.